docs/src/main/paradox/docs/releases/v1.7-to-v1.8-migration.md

```diff
@@ -0,0 +1,447 @@
+
+When ready, create a materialized view on the messages table on the keyspace `delta_1_5`:
+
+```sql
+CREATE MATERIALIZED VIEW IF NOT EXISTS delta_1_5.ordered_messages AS
+    SELECT ser_manifest, ser_id, event
+    FROM delta_1_5.messages
+    WHERE timebucket is not null
+    AND persistence_id is not null
+    AND partition_nr is not null
+    AND sequence_nr is not null
+    AND timestamp is not null
+    PRIMARY KEY(timebucket, timestamp, persistence_id, partition_nr, sequence_nr)
+    WITH CLUSTERING ORDER BY (timestamp asc);
+```
+
+The materialized view will take some time to build depending on the number of events.
+
+### Check the materialized view
+
+These operations may take a while to complete depending on the number of rows in the messages table.
+
+* On the messages table:
+
+```shell
+cqlsh -e 'select timebucket from delta_1_5.messages' | grep rows
+```
+
+* On the materialized view:
+
+```shell
+cqlsh -e 'select timebucket from delta_1_5.ordered_messages' | grep rows
+```
+
+When the materialized view has the same number of rows as the table then it is ready for migration.
+
+## Prepare PostgreSQL
+
+### Create migration tables
+
+Run the following command in the PostgreSQL instance to create the temporary tables needed for the migration:
+
+```sql
+CREATE TABLE IF NOT EXISTS public.migration_offset(
+    name         text        NOT NULL,
+    akka_offset  text        NOT NULL,
+    processed    bigint      NOT NULL,
+    discarded    bigint      NOT NULL,
+    failed       bigint      NOT NULL,
+    instant      timestamptz NOT NULL,
+    PRIMARY KEY(name)
+);
+
+CREATE TABLE IF NOT EXISTS public.ignored_events(
+    type            text        NOT NULL,
+    persistence_id  text        NOT NULL,
+    sequence_nr     bigint      NOT NULL,
+    payload         JSONB       NOT NULL,
+    instant         timestamptz NOT NULL,
+    akka_offset     text        NOT NULL
+);
+```
+
+### Create the PSQL tables
+Run the commands from the @link:[Delta 1.8.x schema.ddl file](https://github.com/BlueBrain/nexus/tree/v1.8.x/delta/sourcing-psql/src/main/resources/scripts/schema.ddl) to create the tables necessary for Delta operations.
+
+## Migration configuration
+
+### Pull the docker image for Delta 1.8
+
+Pull the docker image for Delta 1.8 from @link:[Docker Hub](https://hub.docker.com/r/bluebrain/nexus-delta)
+
+```shell
+docker pull bluebrain/nexus-delta:1.8.x
+```
+
+### Configure Delta in migration mode
+
+This section defines the mandatory configuration to run Delta in migration mode.
+
+### Environment variables
+
+The following environment variables need to be set when launching Delta v1.8 in migration mode:
+
+* `KAMON_ENABLED`: false (not mandatory but recommended)
+* `MIGRATE_DATA`: true
+* `DISABLE_INDEXING`: true
+* `MIGRATION_CONF`: /path/to/migration.conf
+* `DELTA_PLUGINS`: /opt/docker/plugins/
+
+### JVM parameters:
+
+* Adopt the same values as the one for your current deployment
+
+### Java properties
```

Comment:
Some of them are handled by the migration.conf no ?

Reply:
I removed them from the table but added comments to the `migration.conf`